### ISSUE TITLE
Freeze graph and use as a library

### ISSUE THREAD

**Issue by raymond30031** [matterport/Mask_RCNN#1056]
Hello All,
I am new to tensorflow and python.
I would like to use this mask rcnn as a .so or some sort of library that i can call the inference function in c++. For example, the default coco weights would be a good starting point.

I know with pure tensorflow it is possible to freeze graph.
Is this possible for this implementation of Mask_RCNN with keras, numpy, and potential other libraries?
Does anyone have any pointer or tutorial on how to do this for this version of Mask RCNN?

Thanks,
JC

**Comment by fastlater** [comment]
@raymond30031  You dont need all the functions to do inference in other programming language. You just need to load your model and use C++ to read your model. Then, you just need to input the expected input data and then you will get the results of the outputs of this model. you can start checking other issues, especially #218

**Comment by raymond30031** [comment]
Thanks, I will take a look and try it out

**Comment by ChauncyFr** [comment]
@fastlater I understand what you mean, but how do the three input parameters required for this model get in C++?
